wordpress/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample WordPress plugins
+
+WordPress plugins demonstrating various approaches on how to leverage Pantheon Edge Integrations personalization features.
+
+## Description
+
+Each subdirectory is meant to be an individual plugin. If you wish to install any of these plugins on your site, clone the repo locally and move each subdirectory to your `wp-content/plugins` directory.
```

Comment:
We can leave this for now, but  my initial reaction is that there will likely be more than just plugin code in here in the future, including bespoke code snippets unattached to a plugin and possibly themes or theme code. We may want to break this up into subdirectories to reorganize at a later date.

As we only have plugins now, though, I think this is fine.